lib/FtpContext.js

```diff
@@ -288,10 +288,10 @@ exports.FTPContext = class FTPContext {
             if (this._closingError) {
                 // This client has been closed. Provide an error that describes this one as being caused
                 // by `_closingError`, include stack traces for both.
-                const err = new Error(`Client has been closed: ${this._closingError.message}`);
-                err.stack += `\nCaused by: ${this._closingError.stack}`;
+                const err = new Error("Client is closed");
+                err.stack += `\nClosing reason: ${this._closingError.stack}`;
```

Comment:
Does `stack` include the  message, or should `message` be included here as well?

Reply:
It includes the message, that's why I left it away.